src/python/ensembl/io/genomio/fasta/process.py

```diff
@@ -96,6 +99,18 @@ def main() -> None:
     parser.add_argument_src_path("--genbank_infile", help="Input GenBank GBFF file")
     parser.add_argument_dst_path("--fasta_outfile", required=True, help="Output FASTA file")
     parser.add_argument("--peptide_mode", action="store_true", help="Process proteins instead of DNA")
+    parser.add_log_arguments(add_log_file=True)
     args = parser.parse_args()
 
-    prep_fasta_data(**vars(args))
+    init_logging(
+        args.log_level,
+        log_file=args.log_file,
+        log_file_level=args.log_file_level,
+    )
```

Comment:
We have this simpler version now:
```suggestion
    init_logging_with_args()
```